cloudprem/eks.tf

```diff
@@ -66,10 +66,7 @@ data "aws_iam_policy_document" "eks_worker" {
       "s3:PutObject",
       "s3:PutObjectAcl",
       "s3:GetObject",
-      "s3:GetObjectAcl",
-      "s3:ListBucket",
-      "s3:ListObjectsV2",
-      "s3:CopyObject"
```

Comment:
Woah. I'm not sure i grok whats happening here. Does this `data` block fetch the AWS default policy for the `eks_workers`? If so, why do we need to specify actions for a `data` source? I'm a little confused by this...

Reply:
Yep. These got put right back in: https://github.com/Dozuki/CloudPrem-Infra/pull/3/commits/0bcf116cda2e91d0223998e4eadf75724eec3b9e